packages/api-rest/package.json

```
@@ -47,7 +47,7 @@
 		"src"
 	],
 	"dependencies": {
-		"axios": "1.5.0",
+		"axios": "0.26.0",
 		"tslib": "^2.5.0",
 		"url": "0.11.0"
```

We'll need to remove this dependency at some point to avoid introducing another install warning.

```suggestion
		"url": "0.11.0"
```